### DISCUSSION TITLE
android apps crash with latest air sdk's

### DISCUSSION THREAD

**Discussion by mehmury** [airsdk/Adobe-Runtime-Support#1904] [General]
I tried to update my apps with air sdk version 33.1.1.856.
Applications that I compile as aab or apk with Adobe Animate crash when I install them on my android device.
The anes I use are firebase and admob from myflashlabs.
In addition, I also use the discriqt IDFA ane.
In versions after Air SDK's version 33.1.1.743,(in 33.1.1. 795,33.1.1.821,33.1.1.856) after compiling applications on the android side, it does not work on the device.
I tried using both java JDK 17.0.1 and java JDK 18.0.1.1, but my apps crash on android devices.
I hope the same problem does not continue in new sdk versions.

**Comment by ajwfrost** [comment]
Hi - can you provide a logcat output of what's happening on the device?
With 33.1.1.856, if you have the device connected via USB, it should be possible to run
```
adt -deviceLog -platform android
```
and get the output (you can redirect by adding `> output.log` after that command line, most likely you'll need to hit ctrl-c a couple of times in order to close the process)

thanks

**Reply by mehmury** [reply]
I noticed it while reviewing a few previous forum threads.
I have read that multidex support is removed in recent sdk versions.
While checking the old codes required for firebase in the main.xml content, I noticed that multidex support is turned on.


`<application android:name="android.support.multidex.MultiDexApplication">`

I changed THIS LINE to `<application>` and now when I try again it works fine with 33.1.1.856 my applications.
thanks HARMAN...and @ajwfrost
